doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -366,3 +374,34 @@ otherwise if `false`:
 ==== Schema
 
 If APOC are installed, schema will be created with `apoc.meta.relTypeProperties`. Otherwise the first 10 (or any number specified by the `schema.flatten.limit` option) results will be flattened and the schema will be create from those properties.
+
+=== Read data by custom Cypher Query
+
+You can specify a Cypher query in this way:
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+      .option("query", "MATCH (n:Person) WITH n LIMIT 2 RETURN collect(n) AS nodes")
+      .load()
+
+df.show()
+```
+
+The struct of the Dataset returned by the query is influenced by the query itself, in this particular context it could happen
+that the connector could not be able to sample the Schema from the query, in these particular cases we suggest trying with
+the option `schema.strategy` defined as `string` as it follows:
+
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+      .option("query", "MATCH (n:Person) WITH n LIMIT 2 RETURN collect(n) AS nodes")
+      .option("schema.strategy", "string")
+      .load()
+
+df.show()
+```
+
+This means that the struct returned by the query will be composed by strings that you can than cast via simply Spark's
+transformations.
```

Comment:
add note.   Inference is good when all instances of a property in neo4j are the same type, and string followed by cast is better when property types may differ.  Remember that Neo4j does not enforce property typing, and so person.age could sometimes be a long and sometimes be a string.

Reply:
Done